demos/common/cpp/pipelines/src/async_pipeline.cpp

```diff
@@ -96,14 +101,16 @@ int64_t AsyncPipeline::submitData(const InputData& inputData, const std::shared_
                     }
                 }
             }
+
             condVar.notify_one();
-    });
+        });
```

Comment:
```suggestion
    });
```
according to verbally-defined code style, closing bracket should have same indentation as start of opening expression (not opening bracket, as it should be at the end of line).
BTW, look like whole operators block has extra intentation and hould be shifted to the left by one tab